### DISCUSSION TITLE
Suggestions on how we will call `break` and `continue` loop statements in Kin

### DISCUSSION THREAD

**Discussion by pacifiquem** [kin-lang/kin#146] [Ideas]
We're planning to add `break` and `continue` loop control structures in (#70) and as y'all know, Kin is in Kinyarwanda so if you've any idea on how we can call those, please comment it! Get your credits in our next release on 7th August.

**Comment by ndungtse** [comment]
I think `continue` can be called `komeza` then `break`  be `hagarika` or `hagarara`.

**Reply by pacifiquem** [reply]
Continue => Komeza ✅ 
Break => Hagarara ✅ 

This can work @ndungtse, thanks!
